extendables/channelname.js

```diff
@@ -0,0 +1,34 @@
+const { Extendable } = require('klasa');
+const { GuildChannel } = require('discord.js');
+const channelRegex = new RegExp(/^(?:<#)?(\\d{17,19})>?$/);
+module.exports = class extends Extendable {
+
+	constructor(...args) {
+		super(...args, ['ArgResolver'], { klasa: true });
+	}
+
+	async extend(arg, currentUsage, possible, repeat, msg) {
+		if (arg instanceof GuildChannel) return arg;
+		const matches = channelRegex.exec(arg);
```

Comment:
- What about you test that and wrap the code in an if. Then correctly handle valid/invalid ID?
- And you can use `return this.channel(...);`